akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -2779,17 +2779,43 @@ trait FlowOps[+Out, +Mat] {
    *
    *   '''Backpressures when''' downstream backpressures
    *
-   *   '''Completes when''' any of the upstreams completes
+   *   '''Completes when''' any upstream completes if `eagerComplete` is enabled or wait for all upstreams to complete
```

Comment:
This method signature doesn't have the `eagerComplete` parameter.
```suggestion
   *   '''Completes when''' any of the upstreams completes
```